backstop/tests/date-picker.html

```diff
@@ -1504,74 +1504,75 @@ <h2>Date & time</h2>
             </div>
           </div>
         </div>
+        <div class="iui-time-picker">
```

Comment:
Does it need to have it's own class?  We have `.iui-date-picker` when showing a calendar & a calendar + time combo.  Why not reuse `.iui-date-picker` and only show time within it?

Reply:
Ah wait, I see.  So above we have:
```
<date picker>
  <calendar />
  <time picker />
</date picker>
```

You're proposing:
```
<time picker />
```
where it is not nested within date picker at all?  So you're not really introducing a new class?

Reply:
I have a new wrapper class, which has few styles taken from Date picker to have same styling..
Unless you figure out the way not to have that class (but I think we need it).

Reply:
I am able to pull of something complex like this (not suggesting this layout, just showing how complex it can get) and it can be as simple as a single time column.

![Screen Shot 2021-05-28 at 12 10 32 PM](https://user-images.githubusercontent.com/849817/120012602-b44b1900-bfad-11eb-8485-b2b814a28c40.png) ![Screen Shot 2021-05-28 at 12 11 07 PM](https://user-images.githubusercontent.com/849817/120012665-ca58d980-bfad-11eb-8264-4974e3325b9c.png)

Reply:
Have in mind, that the code we write for TimePicker (as single component) should be put under date picker and it should work.
Will this html work then after your changes??
<iui-date-picker>
....
 <iui-date-picker>
   <iui-time>
   ....
   </iui-time>
 </iui-date-picker>
</iui-date-picker>

?

Reply:
I have moved back separate time picker...